### ISSUE TITLE
path alias doesn't work as expected: Cannot find module

### ISSUE THREAD

**Issue by fxdave** [Igorkowalski94/eslint-plugin-project-structure#26]
My folder structure:
 - datasrc
   - ...
 - frontend
   - src
   - tsconfig.json
   - ...

In the tsconfig:
```json
    "paths": {
       "@datasrc/*": ["../datasrc/*"],
     }
```

After digging in the code, I found it removes the ../ intentionally. To me it seems it should just keep it as it is defined.
See here:
https://github.com/Igorkowalski94/eslint-plugin-project-structure/blame/11df61fa1e30ee55487665d3c1cfb31945a55731/src/rules/independentModules/helpers/validateImport/helpers/validateAll/helpers/getImportPaths.ts#L26

Rolling back to 3.7.1 solved the issue.

**Comment by fxdave** [comment]
I made a workaround https://github.com/fxdave/eslint-plugin-project-structure
However, this is far from the ideal solution. It still doesn't work exactly how typescript resolves the files.

**Comment by Igorkowalski94** [comment]
Hi @fxdave,

I've introduced fixes in version [3.10.8](https://github.com/Igorkowalski94/eslint-plugin-project-structure/releases/tag/v3.10.8) please check if everything works as expected.

If the import is from outside the project root, you need to write it like this:

Folder structure:
 - datasrc
   - ...
 - frontend
   - src
   - node_modules
   - independentModules.mjs
   - tsconfig.json
   - ...

In the tsconfig:
```json
    "paths": {
       "@datasrc/*": ["../datasrc/*"],
     }
```

```js
   {
      name: "Some module",
      pattern: "src/hello/**",
      allowImportsFrom: [
        "**/datasrc/**/*.ts", // Import from outside the project root.
        "src/helpers/**"      // Import inside the project root.
      ],
    },
```

**Comment by fxdave** [comment]
It works like a charm, Thanks for the quick fix!
